src/Adapter/Presenter/Product/ProductLazyArray.php

```diff
@@ -252,9 +265,13 @@ public function getCondition()
     public function getDeliveryInformation()
     {
         if ($this->product['quantity'] > 0) {
-            return Configuration::get('PS_LABEL_DELIVERY_TIME_AVAILABLE', $this->language->id);
+            $config = $this->configuration->get('PS_LABEL_DELIVERY_TIME_AVAILABLE');
+
+            return $config[$this->language->id] ?? ($config[0] ?? null);
```

Comment:
I don't think `$config[0]` could ever contain a value? 🤔

Reply:
False. In classes/Configuration, if the languageId is not defined, it's defined to 0.

Reply:
I'm maybe missing something but you are using `src/Adapter/Configuration` and when the configuration key is a lang key, [`$this->getLocalized()`](https://github.com/PrestaShop/PrestaShop/blob/66f4b476cb28c82da77b1b64228764ac3a6de7fc/src/Adapter/Configuration.php#L336-L345) is called and I don't see how it can return an array with the key `0`.

Reply:
```suggestion
            return $config[$this->language->id] ?? (reset($config) ?? null);
```
This way you are sure

Reply:
Approved @jolelievre

Reply:
Fixed. Thanks @atomiix & @jolelievre

Reply:
@Progi1984 @jolelievre I'm not sure about that. The first array element could be any langage, not necessarily the default one and I don't think it would make sense to use a "random" lang as fallback ^^

Reply:
@atomiix You're right. I broke the initial behavior. If the lang is not found, i sent null.